token.js

```diff
@@ -56,7 +57,7 @@ class Akamai_EdgeAuth_Config {
 
     set_start_time(start_time) {
         // verify starttime is sane
-        if (start_time.toLowerCase() === "now") {
+        if (start_time.toLowerCase && start_time.toLowerCase() === "now") {
```

Comment:
potato, potahto... or stag, stayg?
```suggestion
        if (typeof start_time === 'string' && start_time.toLowerCase() === 'now') {
```